app/assets/javascripts/calls.coffee

```diff
@@ -39,7 +39,7 @@ jQuery ->
     )
 
     # make the columns of all of the tables sortable
-    autosortable_tables = [ '#call_list table', '#completed_calls table', '#urgent_patients table' ]
+    autosortable_tables = [ '#call_list table', '#completed_calls table', '#urgent_patients table', '#user-list table']
```

Comment:
should this be in a shared .js file, users.coffee? It certainly does not belong here.